### ISSUE TITLE
background images out of size on iOS device

### ISSUE THREAD

**Issue by ElFishi** [tranduyhung/grav-theme-big-picture#29]
I am struggling with the display of the big-picture theme on my iOS devices. 
My own installation of Big Picture [v1.4.2] under Grav v1.7.27.1 always has the problem that the background pictures only show a small corner of the pictures but massively enlarged when viewed on my iOS devices (iPhone and iPad).
![IMG_D1F039049110-1](https://user-images.githubusercontent.com/29286528/150509113-91400fd8-cd88-4d3e-9f59-997c4b18604c.JPEG)
![IMG_6F6E1293CB96-1](https://user-images.githubusercontent.com/29286528/150509125-40592436-fa54-4b87-af1e-5d329483f3d5.JPEG)
From a PC everything looks OK.
Interestingly, the demo  at [grav-theme-big-picture.flatfiledeveloper.com](https://grav-theme-big-picture.flatfiledeveloper.com) works OK on my iPhone, but shows the same behaviour on my iPad (screenshots above). 

Can this be healed? 

Thx

**Comment by SomeonegetaDev** [comment]
Probably a IOS retina media query that can be added to the CSS sheet to the .intro and .info 
 Here is a good article on retina media queries

https://css-tricks.com/snippets/css/retina-display-media-query/

**Comment by jfischetti22** [comment]
Agree.  There is an issue with the Theme correctly sizing the background images in the theme elements.   There is a new version as of 1/20/23, but that version has not fixed the iPad  issue.   Note: I only have the issue on the iPad, so if you upgrade it might fix your iPhone issues at least.
